_posts/blog/http/2018-11-08-napetitions-review.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: "Authorship & Advocacy: A Review of The Native American Petitions Dataverse "
+modified:
+categories: blog
+excerpt: "Embedded in the (digital) archive are structures of power. [The Native American Petitions Dataverse](https://dataverse.harvard.edu/dataverse/nativeamericanpetitions) shifts those structures by attribute both tribal and individual Native authorship to hundreds of colonial and early American petitions and memorials.
+tags: [authorship, petitions, gender, legislation, indigeneity, advocacy]
+author: alinascott
+image: alinascott.jpg
+  feature:
+---
+[Native American Petitions Dataverse](https://dataverse.harvard.edu/dataverse.xhtml?alias=nativeamericanpetitions) Accessed November 1, 2018.   
+*Reviewed by Alina Scott, Department of History. The University of Texas at Austin.*   
+
+**Introduction**  
+Embedded in the (digital) archive are structures of power. [The Native American Petitions Dataverse](https://dataverse.harvard.edu/dataverse/nativeamericanpetitions) shifts those structures by attribute both tribal and individual Native authorship to hundreds of colonial and early American petitions and memorials. However, is attributing authorship the sole responsibility of those engaging with digital collections? And even more simply, how does one acknowledge Indigenous authorship in the colonial and early American archive? Jane Anderson addresses this in part by saying “wherein colonialism is understood as a cultural project of control, archives function as the locus for the cultural technology of rule” (234). By Anderson's definition, the digital structure of [The Native American Petitions Dataverse](https://dataverse.harvard.edu/dataverse/nativeamericanpetitions) is, in part, a decolonial project.   
```

Comment:
The term 'decolonial' here is vague. I offer two recommendations:
1. In the first sentence, be more specific about the kinds of power you're interested in so that we can understand what, exactly, is transformative about the NAPD. Without knowing how petition authorship is normally attributed, it's hard to understand what this project seeks to accomplish.
2. In the final sentence of the paragraph, be more specific in your language. "in part, a decolonial project that does [x] in the archive."